Description: Set the version of cglib in the cglib-nodep pom. This is required to package
 cglib-nodep without a parent pom and separate it from cglib.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/cglib-nodep/pom.xml
+++ b/cglib-nodep/pom.xml
@@ -100,6 +100,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>cglib</artifactId>
+            <version>${project.version}</version>
             <optional>true</optional>
         </dependency>
     </dependencies>
